### ISSUE TITLE
[Bug]

### ISSUE THREAD

**Issue by openlayers-droid** [keplergl/kepler.gl#1467]
**Describe the bug**
An error in deck.gl: Cannot read property '3' of undefined in lirko2f when choosing Grid layer

**To Reproduce**
1. Go to 'import csv'
2. Click on 'import'
3. choose Grid layer
4. toggle Height
5. choose 'Height based on'
6. enter field - int range
7. a red notification box 'An error in deck.gl: Cannot read property '3' of undefined in lirko2f'

**Expected behavior**
display grid as in on keplergl.com/demo

**Screenshots**

**Desktop (please complete the following information):**
 - OS: OSx
 - Browser: all browsers

**Additional context**
Using the same dataset, it works as expected on https://kepler.gl/demo

**Comment by macrigiuseppe** [comment]
@openlayers-droid can you please provide a sample of your dataset (csv file)?

**Comment by openlayers-droid** [comment]
hi @macrigiuseppe 

many thanks for the response. What i found was the same file works on the hosted version. But when i download and build locally it behaves like above.  i will create a subset of the csv file and attach it here.

**Comment by ibgreen** [comment]
@igorDykhta This may be related to one of the issues you are working on?

**Comment by openlayers-droid** [comment]
@igorDykhta it happens only on localhost

**Comment by ibgreen** [comment]
@igorDykhta of course. What I meant is that you are about to overhaul the height handling of a number of layers in deck.gl.
